### DISCUSSION TITLE
Connecting to Master server

### DISCUSSION THREAD

**Discussion by Wythmer** [R2Northstar/Northstar#424] [Tech Support]
![image](https://user-images.githubusercontent.com/121660957/210044608-a35ae681-e772-4635-875c-ec323c02e74e.png)
![image](https://user-images.githubusercontent.com/121660957/210044620-aa6fb688-8261-4d5f-af3b-dee861297aa6.png)
Hi. Can't slove this issue for a long time. What should I do to solve it?

Using Northstar launcher in game directory I have LSX Authentication failed. Can't fix it too

**Comment by GeckoEidechse** [comment]
Does vanilla Titanfall2 work? Also make sure that EA App or Origin is running before you launch Northstar.

**Comment by Wythmer** [comment]
Vanilla TF2 work only when Northstar does not exist in game directory. No matter what I have in settings (-vanilla or -northstar) it'll run Northstar client anyway
Starting Titanfall 2 from Steam I have this error. Starting Titanfall 2 from Northstar_Client.exe I have LSX Authentication failed
Yes, Origin is running before I launch Northstar

**Comment by Wythmer** [comment]
> Does vanilla Titanfall2 work? Also make sure that EA App or Origin is running before you launch Northstar.

Installed EA App. Northstar still doesn't work

**Comment by GeckoEidechse** [comment]
>  No matter what I have in settings (-vanilla or -northstar) it'll run Northstar client anyway

Do you have a file called `run_northstar.txt` in your Titanfall2 folder? Also you might have the `-northstar` still set on Steam/Origin, i.e. the other one compared to the one you checked.
